cloudformation_templates/aws_kibana.json

```diff
@@ -0,0 +1,63 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Description": "Kibana AWS Elasticsearch domain with CloudWatch streaming",
+  "Parameters": {
+    "DomainName": {
+      "Type": "String",
+      "Description": "Domain name for AWS Elasticsearch service"
+    },
+    "UserIPs": {
+      "Type": "CommaDelimitedList",
+      "Description": "List of IP addresses that can access the Elasticsearch domain"
+    }
+  },
+
+  "Resources": {
+    "ElasticsearchDomain": {
+      "Type": "AWS::Elasticsearch::Domain",
+      "Properties": {
+        "DomainName": {"Ref": "DomainName"},
+        "EBSOptions": {
+          "EBSEnabled": true,
+          "VolumeSize": 15,
+          "VolumeType": "gp2"
+        },
+        "ElasticsearchClusterConfig": {
+          "InstanceType": "t2.micro.elasticsearch",
+          "InstanceCount": 1
```

Comment:
We do need CloudWatch – it's where instances send logs to begin with, so streaming is for CloudWatch to Kibana. Also it's a more robust permanent storage than an ES cluster, so the idea is to keep only a few weeks of recent data in Kibana to simplify debugging/monitoring.